.github/workflows/nvda-test.yml

```diff
@@ -0,0 +1,190 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: nvda-test
+on:
+  workflow_dispatch:
+    inputs:
+      aria_at_ref:
+        description: |
+          The sha / ref to checkout for aria-at tests
+        required: false
+        type: string
+      test_pattern:
+        description: |
+          The test pattern passed to the aria-at-automation harness.
+          Uses a default of "{reference/**,test-*-nvda.*}" if not provided
+        required: false
+        type: string
+      work_dir:
+        description: |
+          The --plan-workingdir passed to the aria-at-automation harness.
+          Based from the aria-at repo build folder.
+          Uses a default of tests/alert if not provided.
+        required: false
+        type: string
+      callback_url:
+        description: |
+          The harness will send POST requests to this url using callback_header
+          header with test results.
+        required: false
+        type: string
+      status_url:
+        description: |
+          The harness will send POST requests to this url using callback_header
+          header with status updates.
+        required: false
+        type: string
+      callback_header:
+        description: |
+          The harness will send POST requests to callback_url using this header
+          with status updates.
+        required: false
+        type: string
+      browser:
+        description: |
+          The browser to use for testing, "chrome" or "firefox", default "chrome"
+        required: false
+        type: string
```

Comment:
[I'm on record as being default-averse](https://github.com/w3c/aria-at-app/pull/831#discussion_r1395022090), but I'm trying to keep an open mind. Is there a situation where omitting the browser name would help?

Reply:
Quick manual test run still does alert on a browser without having to provide any options

Reply:
I'll take it!